ui/src/backoffice/containers/DetailPageContainer/AuthorDetailPageContainer.tsx

```diff
@@ -114,7 +115,11 @@ const AuthorDetailPageContainer: React.FC<AuthorDetailPageContainerProps> = ({
                         status === 'error' ? 'white' : ''
                       } w-100`}
                     >
-                      <p className="b f3 tc pv2">{status?.toUpperCase()}</p>
+                      <p className="b f3 tc pv2">
+                        {status && getWorkflowStatusInfo(status.toLowerCase())
```

Comment:
To make it more readable, you can extract this part before the return with the other variables. You can have something like 

```
const statusInfo = status ? getWorkflowStatusInfo(status.toLowerCase()) : null;

```
and then here you just do 
```
{statusInfo ? statusInfo.text : 'Unknown Status'}
```